examples/spring-boot/src/main/java/io/opentracing/contrib/jaxrs2/example/spring/boot/JerseyConfig.java

```diff
@@ -23,13 +22,9 @@
     @Inject
     public JerseyConfig(Tracer tracer) {
         Client client = ClientBuilder.newClient();
+        client.register(new ClientTracingFeature(new Builder(tracer)));
```

Comment:
You found it :+1: , see https://github.com/opentracing-contrib/java-jaxrs/pull/26#discussion_r115731035. I will make that constructor private.